Sensus.UiTest/app.config

```diff
@@ -0,0 +1,15 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <runtime>
+    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
+      <dependentAssembly>
+        <assemblyIdentity name="Newtonsoft.Json" publicKeyToken="30ad4fe6b2a6aeed" culture="neutral" />
+        <bindingRedirect oldVersion="0.0.0.0-9.0.0.0" newVersion="9.0.0.0" />
+      </dependentAssembly>
+      <dependentAssembly>
+        <assemblyIdentity name="PCLCrypto" publicKeyToken="d4421c8a4786956c" culture="neutral" />
+        <bindingRedirect oldVersion="0.0.0.0-2.0.0.0" newVersion="2.0.0.0" />
+      </dependentAssembly>
+    </assemblyBinding>
+  </runtime>
+</configuration>
```

Comment:
That's funny. Your note on the commit specifically says, "Forgot to add app.config file", as if you went back and intentionally added it. I can't even begin to count the number of times I've found code changes and thought, "who did that?", only to discover it was me -- sometimes making the changes as recently as in the last few weeks. Your changes were at least made a year and a half ago.

Yeah I think in our case it makes sense for us to have these because we have a high number of cross dependencies in this project. By that I mean many of our referenced dll's actually bind to one another. Any one project can only bind to one version of a dll at a time (as far as I know) so our code will always bind to the latest version. However, if we update what our code binds to but one of our references still expects to bind to the old version app.config should handle that gracefully.